deploy/openshift/templates/multi/postgres-template.yaml

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2012-2018 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: che
+  annotations:
+    description: Che
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    creationTimestamp: null
```

Comment:
Why do we need it?

Reply:
Removed

Reply:
Where is it removed? I still see few `creationTimestamp` fields in this template. Do I have an issue with browser cache?